addons/project_timesheet_holidays/tests/test_employee.py

```diff
@@ -103,6 +103,14 @@ def test_write_employee(self):
         self.assertEqual(str(timesheet.date), '2020-01-01', 'The timesheet should be created for the correct date')
         self.assertEqual(timesheet.unit_amount, 8, 'The timesheet should be created for the correct duration')
 
+        # test unarchiving on an already active employee does not create duplicate public leaves
```

Comment:
For reference this happen here: https://github.com/odoo/enterprise/blob/e6bb2a0a277c24873f2ab7fb37bfb2e77b52373f/hr_contract_salary/controllers/main.py#L59